lib/tasks/add_navigation_links.rake

```diff
@@ -52,13 +54,24 @@ namespace :navigation_links do
   end
 
   namespace :find_or_create do
+    task home: :environment do
+      NavigationLink.where(url: "/").first_or_create(
+        name: "Home",
+        url: URL.url("/"),
+        icon: home_icon,
+        display_only_when_signed_in: false,
+        position: 0,
+        section: :default,
+      )
+    end
+
     task readinglist: :environment do
       NavigationLink.where(url: "/readinglist").first_or_create(
         name: "Reading List",
         url: URL.url("readinglist"),
         icon: reading_icon,
         display_only_when_signed_in: true,
-        position: 0,
+        position: 1,
```

Comment:
I think changing the position could have unexpected problems.

The `first_or_create` method uses the provided attributes as the "WHERE" clause of a SQL statement.  If it finds on those conditions, it uses that.  Otherwise it creates the record.

This could mean we could have two "/readinglist" links at position 0 and 1.

My proposal as a stop-gap is to set the "/" position to -1.

Reply:
Ah yes, good point. -1 sounds like a good stop-gap for now

Reply:
The only thing about that approach is that in the Admin tools, a user will see -1 as the position 😕 

![navigation links view with Home at -1 position](https://user-images.githubusercontent.com/20773163/150802239-0b7a16dc-2f8b-4eb2-883c-67cac7655cee.png)

I'm not sure how much of a problem that is

Reply:
We can update the view code to do `position + 2`, most normal people start counting from 1 and not 0 😉

Reply:
> Though I guess that's less than ideal too because then it won't be the same number when editing.

Yeah good point. I guess we don't prohibit negative values for that position field when editing, so maybe it's fine that it shows as -1? I'm tempted to think it's OK... like you say '0' probably seems equally strange to most people 🤔